src/components/attendee/AttendeeLayout.tsx

```diff
@@ -63,6 +83,8 @@ const _AttendeeLayout: React.FC<_AttendeeLayoutProps> = ({ userId }) => {
 export const AttendeeLayout = compose(
   withAuth,
   withFallback(["isAuthLoaded", "userId"], NotLoggedInFallback),
+  withSlugs,
+  withWorldAndSpace,
   withRequired({
     required: ["userId"],
     fallback: LoadingPage,
```

Comment:
[OOS]
There is a nice way of rewriting this. Something in line of https://github.com/sparkletown/sparkle/blob/redesign-2021/src/components/templates/ConversationSpace/ConversationSpace.hoc.tsx